spec/requests/page_list_spec.rb

```diff
@@ -0,0 +1,108 @@
+require "rails_helper"
+
+RSpec.describe "Pages", type: :request do
+  before do
+    User.create!(email: "user@example.com")
+  end
```

Comment:
Why is this needed?

Reply:
Not sure actually - this was just moved over from the existing pages controller tests. I'll see if removing it breaks anything